### PR TITLE
IGVF-608-sub-cellular-pool

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/schemas/biosample.json [hunk 1]
```diff
@@ -219,6 +219,12 @@
             "comment": "See modification.json for available identifiers.",
             "type": "string",
             "linkTo": "Modification"
+        },
+        "cellular_sub_pool": {
+            "title": "Cellular Sub Pool",
+            "description": "Cellular Sub Pool of a sample",
+            "type": "string",
+            "pattern": "^[a-zA-Z\\d_$.+!*,()'-]+(?:\\s[a-zA-Z\\d_$.+!*,()'-]+)*$"
```
Comment (gabdank) [inline src/igvfd/schemas/biosample.json:227]: I think we need a comment providing examples of valid identifiers

File: src/igvfd/schemas/changelogs/biosample.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ## Changelog for *`biosample.json`*
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/biosample.md:2]: i thikn we need to update the changelogs of the concrete classes (tissue, etc.) with minor chnages

**Review by gabdank — CHANGES_REQUESTED** [review]
minor changes requested

File: src/igvfd/schemas/biosample.json (hunk 1 above)
Comment (khzlin) [reply src/igvfd/schemas/biosample.json:227]: ok

File: src/igvfd/schemas/changelogs/biosample.md (hunk 2 above)
Comment (khzlin) [reply src/igvfd/schemas/changelogs/biosample.md:2]: done

**Review by gabdank — APPROVED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**